hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/command/AlterHoodieTableDropPartitionCommand.scala

```diff
@@ -57,6 +55,7 @@ case class AlterHoodieTableDropPartitionCommand(
         sparkSession.sessionState.conf.resolver)
     }
 
+    // drop partitions to lazy clean
```

Comment:
Can you please expand on that comment? I get the general direction from it but it isn't clear what it holistically is referring to